cupy_backends/hip/cupy_hipsparse.h

```diff
@@ -0,0 +1,3748 @@
+
+
+#ifndef INCLUDE_GUARD_HIP_CUPY_HIPSPARSE_H
+#define INCLUDE_GUARD_HIP_CUPY_HIPSPARSE_H
```

Comment:
Optional and to be discussed, but maybe is a good time to start using `#pragma once`

Reply:
I don't have any preference on this. Let me know if the team has any!

Reply:
thanks! will do